tests/phpunit/CRMTraits/Financial/PriceSetTrait.php

```diff
@@ -61,4 +61,6 @@ protected function createContributionWithTwoLineItemsAgainstPriceSet($params, $l
     $this->callAPISuccess('order', 'create', $params);
   }
 
+  protected function
```

Comment:
@eileenmcnaughton this is causing PHPFatal Errors

Reply:
ug

Reply:
yeah lots of issues with what is actually in the commit looking at it